app/src/main/res/drawable/single_player_button.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item>
+        <layer-list>
+            <item android:right="0dp" android:top="0dp">
+                <shape>
+                    <corners android:radius="100dp" />
+                    <solid android:color="@color/SPButtonShadow" />
+                </shape>
+            </item>
+            <item android:bottom="4dp" android:left="0dp">
+                <shape
+                    xmlns:android="http://schemas.android.com/apk/res/android"
+                    android:shape="rectangle">
+
+                    <gradient android:startColor="@color/SPButtonGreenLight"
+                        android:endColor="@color/SPButtonShadow"
```

Comment:
`sp_button_shadow` for variables inside xml.
`aButton` camel case for variables inside Java.